pennylane_lightning/core/src/simulators/lightning_kokkos/measurements/MeasurementsKokkos.hpp

```diff
@@ -509,117 +513,68 @@ class Measurements final
      */
     auto probs(const std::vector<std::size_t> &wires)
         -> std::vector<PrecisionT> {
-        PL_ABORT_IF_NOT(
-            std::is_sorted(wires.cbegin(), wires.cend()),
```

Comment:
Unsorted wires supported.

Reply:
Neat! @LuisAlfredoNu, keep your eyes on that. This will affect your work.